pf9/cluster/commands.py

```diff
@@ -388,7 +390,16 @@ def attach_node(ctx, **kwargs):
 
     # There may be 0 masters, workers specified. The attach_cluster method
     # handles this situation
-    attach_cluster(ctx.params['cluster_name'], master_ips, worker_ips, ctx)
+    try:
+        attach_cluster(ctx.params['cluster_name'], master_ips, worker_ips, ctx)
+    except CLIException as e:
+        click.secho("Encountered an error while attaching nodes to a Kubernetes"\
+                    " cluster {}. {}".format(ctx.params['cluster_name'], e.msg), fg="red")
+        sys.exit(1)
+    else:
```

Comment:
nit: don't see the need for an `else` here. Just un-indent the next `click.secho`?

Reply:
Addressed this in a couple of places where I found similar code pattern